recipes/plasmapy/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "plasmapy" %}
+{% set version = "0.1.dev841" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "1689b1d5a773543648aa29ef6ea1ab5c2fa94c66e82b972a37cac51a620ca71f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record=record.txt
```

Comment:
Do you need the `--offline` and `--no-git`?

Reply:
I thought I did, but I'll try it without again and see if it builds.

Reply:
Turns out I do need the `--offline` but not the `--no-git`, so I'll remove that.